plugins/woocommerce/client/blocks/assets/js/blocks/payment-methods/style.scss

```diff
@@ -0,0 +1,37 @@
+.wp-block-woocommerce-payment-methods {
+	.wc-block-payment-methods__list {
+		list-style: none;
+		margin: 0;
+		padding: 0;
+		display: flex;
+		flex-wrap: wrap;
+		gap: 12px;
+	}
+
+	.wc-block-payment-methods__list-item {
+		display: flex;
+		align-items: center;
+		justify-content: center;
+		min-height: 24px;
+		padding: 12px;
+		border: 1px solid currentColor;
+		border-radius: 4px;
```

Comment:
Should we just use the button element class instead to let the block inherit from the theme? maybe use the outline class if needed too

Reply:
I ended up removing this border as the icons included in WooPayments already have a border, and I think these are the ones used in the theme design.